src/docker/distributed/server/compose.yml

```
@@ -2,6 +2,10 @@ services:
   superlink:
     image: flwr/superlink:${FLWR_VERSION:-1.12.0}
```

```suggestion
    image: flwr/superlink:${FLWR_VERSION:-1.12.0}
```
Same as [this comment](https://github.com/adap/flower/pull/4413#discussion_r1827899886). This should be `nightly` for now, until `1.13` is released.